src/config/dynamic-theme-fixes.config

```diff
@@ -216,6 +216,19 @@ div.description > span {
 
 ================================
 
+500px.com
+
+CSS
+.button.new_fav .icon .shape {
+    fill: transparent !important;
+    stroke: rgb(232, 230, 227);
```

Comment:
```suggestion
```
The stroke value is already correctly modified by dark reader.

Reply:
Sorry, I lost this value in my tests.